.github/workflows/ci.yml

```diff
@@ -123,14 +138,15 @@ jobs:
 
     - name: Upload web
       run: rsync -az --delete --exclude='pandas-docs' --exclude='docs' --exclude='Pandas_Cheat_Sheet*' web/build/ docs@${{ secrets.server_ip }}:/usr/share/nginx/pandas
-      if: github.event_name == 'push'
+      if: ${{ github.event_name == 'push' && steps.changes.outputs.web == 'true' }}
```

Comment:
The reason for this edit is that I am afraid that because there is a way that we skip the build, so ```rsync``` will sync the local directory with the remote directory and if the local directory is empty then it might delete everything at the server?

I'm not sure, so I added this, will remove if I understood the command wrong.